main.go

```diff
@@ -287,19 +132,19 @@ func installVersion(arg string, custBinPath *string, mirrorURL *string) {
 		installFileVersionPath := lib.ConvertExecutableExt(filepath.Join(installLocation, versionPrefix+requestedVersion))
 		recentDownloadFile := lib.CheckFileExist(installFileVersionPath)
 		if recentDownloadFile {
-			lib.ChangeSymlink(installFileVersionPath, *custBinPath)
+			lib.ChangeSymlink(installFileVersionPath, customBinaryPath)
 			fmt.Printf("Switched terraform to version %q \n", requestedVersion)
 			lib.AddRecent(requestedVersion) //add to recent file for faster lookup
 			os.Exit(0)
 		}
 
 		//if the requested version had not been downloaded before
```

Comment:
While you're here please
```suggestion
		// If the requested version had not been downloaded before
```